rebar.config

```diff
@@ -3,15 +3,15 @@
             {i, [".", "include"]}
            ]}.
 
-{require_otp_vsn, "(^1[89])|^20"}.
+{require_otp_vsn, "(^1[89])|(^2[01])"}.
```

Comment:
if we leave support for 18 shouldn't we have regexp `(^19)|(^2[01])` now ?

Reply:
@ludwikbukowski I update this.